app/assets/javascripts/discourse/routes/group-activity-topics.js.es6

```diff
@@ -9,5 +9,12 @@ export default DiscourseRoute.extend({
     return this.store.findFiltered("topicList", {
       filter: `topics/groups/${this.modelFor("group").get("name")}`
     });
+  },
+
+  actions: {
+    didTransition() {
+      this.controllerFor("application").set("showFooter", true);
```

Comment:
It looks like you can set showFooter to true in a more straightforward manor like this:
```javascript
export default DiscourseRoute.extend({
  showFooter: true,
})
```

That is the pattern in these files just to mention a few
```
./app/assets/javascripts/discourse/routes/group-manage-membership.js.es6:  showFooter: true,
./app/assets/javascripts/discourse/routes/preferences-tags.js.es6:  showFooter: true
./app/assets/javascripts/discourse/routes/preferences-emails.js.es6:  showFooter: true
./app/assets/javascripts/discourse/routes/tag-groups-new.js.es6:  showFooter: true
```